PWGCF/TableProducer/filterCorrelations.cxx

```diff
@@ -326,7 +418,7 @@ struct MultiplicitySelector {
   O2_DEFINE_CONFIGURABLE(cfgCutEta, float, 0.8f, "Eta range for tracks")
 
   Filter trackFilter = (nabs(aod::track::eta) < cfgCutEta) && (aod::track::pt > cfgCutPt);
-  Filter trackSelection = (requireGlobalTrackInFilter()) || (aod::track::isGlobalTrackSDD == (uint8_t) true);
```

Comment:
There's a problem with the clang-format and the subsequent checks, they are not compatible. To pass the test, this needs to be manually changed back to `(uint8_t) true` (with the space) each time before commit.